README.md

```diff
@@ -51,7 +51,7 @@ Pyomo is available under the BSD License - see the
 
 Pyomo is currently tested with the following Python implementations:
 
-* CPython: 3.7, 3.8, 3.9, 3.10, 3.11
+* CPython: 3.8, 3.9, 3.10, 3.11
 * PyPy: 3.7, 3.8, 3.9
```

Comment:
Should we remove `3.7` from the PyPy line too? I don't remember how we got this list of supported PyPy versions, does it map to something in the conda-forge build?